docs/dev/placement-api.md

```diff
@@ -0,0 +1,69 @@
+# Node Placement configuration for Template Validator and Node Labeller
+
+1. The cluster admin can modify the Template Validator/Node Labeller CR to inform what nodes the respective operator pods should be deployed on.
+2. Template Validator pods are placed on nodes that expect infra components according to the HCO API. 
+
+```
+Example TV CR with empty Affinity, Node Selector and Toleration values:
+
+apiVersion: ssp.kubevirt.io/v1
+kind: KubevirtTemplateValidator
+metadata:
+  name: kubevirt-template-validator
+spec:
+  affinity: {}
```

Comment:
Please provide an example (you can use the example we have in the `functests` manifests)

Reply:
Same for NL

Reply:
Done